keras/layers/normalization.py

```diff
@@ -88,6 +87,11 @@ def __init__(self,
         self.beta_constraint = constraints.get(beta_constraint)
         self.gamma_constraint = constraints.get(gamma_constraint)
 
+        if axis == -1 and K.image_data_format() == 'channels_first':
+            self.axis = 1
+        else:
+            self.axis = axis
```

Comment:
"-1" means "last axis", not "feature axis". Please revert.

Reply:
Reverted.

But this again assumes feature axis to be last by default?
Same request as above, is there any way to allow flexibility by recommending 'channels_last' format for MXNet backend? Thereby standardizing the usage?